docs/deploy.md

```diff
@@ -1,71 +1,42 @@
-
-```bash
-    > gcloud components install kubectl
-```
-
-Hopefully you'll manage to walk through the simple setup quickly and have it up and running. The next step would be creating a cluster and setting it up with your application, but since we have already taken care of that, you'll want to run the below command to have your CLI connected to the cluster we have already created so that you can deploy to it. This command will attempt to retrieve the credentials for the 'hackboard' cluster we have set up and allow you to deploy to it.
-
-```bash
-    > gcloud container clusters get-credentials hackboard
-```
-
-## Intalling Docker CE
-
-We will be using the Community Edition of Docker for our application. Head over to [this link](https://www.docker.com/community-edition#download) and choose and specific installation for your own operating system; the setup is rather intuitive and skipped in this guide. Once you've got Docker installed, try running `docker —version` to make sure it's up and running. Note that docker has a 'daemon' service which should be running for the CLI to communicate with. I suggest following the steps in [this page](https://docs.docker.com/install/linux/linux-postinstall/#configure-docker-to-start-on-boot) to make sure the daemon is configured to start with your machine on start-up. Also, the previously mentioned note is mostly directed towards Linux users; the Windows and MacOS application (I believe) takes care of starting up the daemon for you once you see the green light in your indicators.
-
-**NOTE**: **The following steps are optional and meant for you to understand how the deployment actually works. You will not have to (and it is not recommended) to run all these steps manually everytime, as there will be a bash script setup to take care of all this.**
-
-## Building the Docker Image
-
-Every Docker container requires an image to run, which in turn requires a 'Dockerfile' as a set of instructions for creating it. Images can extend off of other images or be created from scratch. Due to the fact that the Docker community is very well established and has an 'almost' exhaustive list of base images on it already, we will be using an official Node JS image that provides the runtime for our application to run, and add to it as we wish. Below is the basic Dockerfile we have created so far:
-
-```bash
-    FROM node:carbon
-    ENV PORT 8080
-    WORKDIR /usr/src/app
-    COPY package*.json ./
-    ADD VERSION .
-    RUN npm install
-    COPY . .
-    EXPOSE 8080
-    CMD [ "npm", "start" ]
+This application can be most easily deployed via Heroku. A "deploy to heroku" button is available in the `README.md` file. Once in Heroku, several environment variables will need to be filled out including: 
+
+```plain
```

Comment:
will this change our `.env.example`?

Reply:
No. Only a subset of that is needed for a production deploy.